package.json

```diff
@@ -7,10 +7,11 @@
   "type": "module",
   "scripts": {
     "build": "tsup",
+    "check-fmt": "prettier --check .",
     "format": "prettier --write .",
-    "lint": "pnpm run prebuild && eslint src/**/*.ts",
+    "lint": "eslint src/**/*.ts",
     "prebuild": "cp package.json ./src/package.json",
-    "all": "pnpm run format && pnpm run lint && pnpm run build"
+    "all": "pnpm run check-fmt && pnpm run lint && pnpm run prebuild && pnpm run build"
```

Comment:
I really like that `all` actually does the formatting, so I can work it in to my workflow really easily. Can we bring it back? 🙏

Reply:
Done